pyodata/client.py

```diff
@@ -44,7 +44,7 @@ def __new__(cls, url, connection, odata_version=ODATA_VERSION_2):
 
             # create model instance from received metadata
             logger.info('Creating OData Schema (version: %d)', odata_version)
-            schema = pyodata.v2.model.Edmx.parse(resp.content)
+            schema = pyodata.v2.model.Edmx.parse(resp.content, namespaces)
```

Comment:
Good job! I think we should switch to the method `schema_from_xml` in another PR ;)